gufe/tests/test_proteincomponent.py

```diff
@@ -278,7 +303,7 @@ def test_hash_neq_name(self, PDB_181L_path):
     def test_protein_total_charge(self, PDB_181L_path):
         m1 = self.cls.from_pdb_file(PDB_181L_path)
 
-        assert m1.total_charge == 7
+        assert m1.total_charge == 20
```

Comment:
This is a large difference in the calculated total charge, can we check where this is coming from, ie what was assigned the wrong formal charge in the old version?